framework/tokens.py

```diff
@@ -37,7 +38,7 @@ def make_token(self, id: str, password_reset: int):
 
         user = await framework.models.User.get(decoded["id"])
 
-        if user is None or decoded["lr"] != user.last_pass_reset or decoded["iat"] < datetime.utcnow():
+        if user is None or datetime.fromisoformat(decoded["lr"]) != user.last_pass_reset:
```

Comment:
I would like to know the rationale why we're using a third-party library for ISO date and times.

Is this because Python is not doing it right?

Reply:
This is part of Python's standard library.